packages/vite/src/node/plugins/resolve.ts

```diff
@@ -923,29 +923,28 @@ export function resolvePackageEntry(
     return cached
   }
   try {
-    let entryPoint: string | undefined | void
+    let entryPoints: string[] = []
```

Comment:
The new `resolveExports` returns an array of possible module paths. It prefers to return an empty array, rather than throw an error. Vite should only if none of the paths in the array are found.